src/electionguard/guardian.py

```diff
@@ -412,7 +420,7 @@ def all_election_partial_key_backups_verified(self) -> bool:
         return True
 
     # Joint Key
-    def publish_joint_key(self) -> Optional[ElementModP]:
+    def publish_joint_key(self) -> Optional[ELECTION_JOINT_PUBLIC_KEY]:
```

Comment:
im of mixed feelings for renaming all of the types from `ElementModP` to descriptive types like `ELECTION_JOINT_PUBLIC_KEY`.  on one hand it improves readability in this code base, on the other hand it is going to take a fair amount of work to make it consistent across all languages.  in any case, there are a number of functions in this file that still return `Tuple[ElementModP,...]` or just `ElementModP` so i'm game for either but i think we should try to fix it in whole files whenever we make the change once in a file.  thoughts?

Reply:
I'm not sure how to tackle this. Perhaps this should be a larger issue. I truly think we need to shift to a recognizable type but I don't think every language should necessarily do this. We should discuss this in detail.